airbyte-cdk/python/airbyte_cdk/utils/schema_inferrer.py

```diff
@@ -81,7 +92,7 @@ def _clean(self, node: InferredSchema) -> InferredSchema:
                 if len(node["anyOf"]) == 2 and {"type": "null"} in node["anyOf"]:
                     real_type = node["anyOf"][1] if node["anyOf"][0]["type"] == "null" else node["anyOf"][0]
                     node.update(real_type)
-                    node["type"] = [node["type"], "null"]
+                    node["type"] = ["null", node["type"]]
```

Comment:
Can we switch the order of these to `[node["type"], "null"]` instead? And also change line 109 and 111 to also put the node type first and `null` second?

The reason is that I think it is clearer in the UI to show the actual field type first and `null` second. See screenshots for comparison:

<img width="892" alt="Screenshot 2024-03-18 at 2 46 56 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/7ab4322a-bd98-44fb-a6da-ac1919a72f73">

<img width="883" alt="Screenshot 2024-03-18 at 4 31 15 PM" src="https://github.com/airbytehq/airbyte/assets/22731524/89734b25-7107-4cf0-8413-740a47c2c522">

Reply:
+1 with the caveat that I don't know if CAT will complain with the re-ordering

Reply:
Updated the order of types and I've added a test in CATs to confirm that